src/web/metrics.rs

```diff
@@ -124,7 +124,7 @@ pub fn metrics_handler(req: &mut Request) -> IronResult<Response> {
                 .get(0),
         );
         PRIORITIZED_CRATES_COUNT.set(
-            ctry!(conn.query("SELECT COUNT(*) FROM queue WHERE priority >= 0;", &[]))
+            ctry!(conn.query("SELECT COUNT(*) FROM queue WHERE attempts < 5 AND priority <= 0;", &[]))
                 .get(0)
                 .get(0),
```

Comment:
This should fix the `rustfmt` test failure:
```suggestion
            ctry!(conn.query(
                "SELECT COUNT(*) FROM queue WHERE attempts < 5 AND priority <= 0;",
                &[]
            ))
            .get(0)
            .get(0),
```